src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Relic-Strings.json

```diff
@@ -15,6 +15,15 @@
       "Cards added: "
     ]
   },
+  "stsjorbsmod:BottledBurdenRelic": {
+    "NAME": "Bottled Burden",
+    "FLAVOR": "I carry this with me, always",
+    "DESCRIPTIONS": [
+      "Upon pickup, choose a card that does not #yExert. Start each combat with that card #yEntombed [REMOVE_SPACE]. #yExhume it on turn %1$s.",
+      "Choose a card for ",
+      "At the start of turn #b%1$s [REMOVE_SPACE], Exhume #y%2$s [REMOVE_SPACE]."
```

Comment:
I just learned about this a week ago or so doing the non-legendary wording. I have it here so that the comma and period are not highlighted. I suppose we can just leave it out for consistency's sake.